src/main/ionic/peers/src/app/app.component.ts

```diff
@@ -1,30 +1,45 @@
 import { Component, ViewChild } from '@angular/core';
-import { Nav, Platform } from 'ionic-angular';
+import { LoadingController, Nav, Platform } from 'ionic-angular';
```

Comment:
I believe loadingcontroller is not used here. Can you optimize the imports? (option+cmd+o is the shortcut on mac, so I think ctrl+alt+o on linux)

Reply:
Ctrl+Shift+Alt+L is to auto-format the code on linux